controllers/utils/constants.go

```diff
@@ -8,5 +8,6 @@ const (
 	PrepPullImages     string = "prepPullImages.sh"
 	PrepSetupStateroot string = "prepSetupStateroot.sh"
 	PrepCleanup        string = "prepCleanup.sh"
-	IBUName                   = "upgrade"
+	IBUName            string = "upgrade"
+	IBUFilePath        string = "/opt/ibu.json"
```

Comment:
nit: This doesn't seem like the right path for this, writing the file to the top-level /opt directory. Why not reuse /var/ibu directory, ie. the IBUWorkspacePath constant?

Reply:
This is for the B side. Do we have anything to carry forward to the B side under /var/ibu?